Symfony/CS/Fixer/PSR2/OneLineMethodArgumentFixer.php

```diff
@@ -0,0 +1,208 @@
+                }
+            }
+        }
+    }
+
+    /**
+     * Fix arguments for given function.
+     *
+     * @param Tokens $tokens             Tokens to handle
+     * @param int    $startFunctionIndex Start parenthesis position
+     * @param int    $endFunctionIndex   End parenthesis position
+     */
+    private function fixFunctionArguments(Tokens $tokens, $startFunctionIndex, $endFunctionIndex)
+    {
+        for ($index = $startFunctionIndex; $index <= $endFunctionIndex; $index++) {
+            $token = $tokens[$index];
+
+            if ($token->equals('(')) {
+                if (!$this->isNewLine($tokens[$index + 1]) && $tokens[$index + 1]->getContent() !== ')') {
+                    $this->addNewLineOn($tokens, $index, $endFunctionIndex);
+                }
+            }
+            if ($token->equals(')')) {
+                if (!$this->isNewLine($tokens[$index - 1]) && $tokens[$index - 1]->getContent() !== '(') {
+                    $this->addNewLineOn($tokens, $index - 1, $endFunctionIndex, '');
+                }
+            }
+
+            if ($token->equals(',')) {
+                $this->fixNewLine($tokens, $index, $endFunctionIndex);
+            }
+        }
+    }
+
+    /**
+     * Inserts new line after comma.
+     *
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $endFunctionIndex
+     */
+    private function fixNewLine(Tokens $tokens, $index, &$endFunctionIndex)
+    {
+        if (
+            $tokens[$index + 1]->isWhitespace()
+            && $tokens[$index + 2]->isComment()
+            && $this->isNewLine($tokens[$index + 3])
+        ) {
+            $index += 3;
+        }
+
+        $this->addNewLineOn($tokens, $index, $endFunctionIndex);
+    }
+
+    /**
+     * Adds new line to the given token.
+     *
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $endFunctionIndex
+     * @param string $indention
+     */
+    private function addNewLineOn(Tokens $tokens, $index, &$endFunctionIndex, $indention = '    ')
+    {
+        $newToken = new Token(array(T_WHITESPACE, "\n".$indention));
+
+        if ($tokens[$index]->isWhitespace()) {
+            $tokens->overrideAt($index, $newToken);
+        } else {
+            if ($tokens[$index + 1]->isWhitespace()) {
+                $tokens->overrideAt($index + 1, $newToken);
+            } else {
+                $tokens->insertAt($index + 1, $newToken);
+                $endFunctionIndex++;
+            }
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $startIndex
+     * @param int    $endIndex
+     *
+     * @return bool
+     */
+    private function hasNewLineBetweenParenthesis(Tokens $tokens, $startIndex, $endIndex)
+    {
+        /** @var Token $token */
+        foreach ($tokens as $index => $token) {
+            if ($index <= $startIndex || $index >= $endIndex) {
```

Comment:
use simple for loop and then you won't need to iterate through whole collection, it would be great performance boost